CHANGES

```diff
@@ -16,6 +16,9 @@ Features added
 Bugs fixed
 ----------
 
+* #6900: Not possible to set :confval:`latex_use_xindy` via ``-D`` option to
+  :program:`sphinx-build`
```

Comment:
2.2.3 should be used severe bug like crashes. So I prefer to use 2.0 branch instead in such case.